admin/users.php

```diff
@@ -66,7 +66,7 @@
       $profilesList = array_merge([['id' => 0, 'text' => TEXT_CHOOSE_PROFILE]], zen_get_profiles());
     } else {
       $action = '';
-      $messageStack->add(SUCCESS_NEW_USER_ADDED, 'success');
+      $messageStack->add(sprintf(SUCCESS_NEW_USER_ADDED, zen_db_prepare_input($_POST['name'])), 'success');

```

Comment:
Since we're doing output here, let's use `zen_output_string_protected` instead of `zen_db_prepare_input`.